lti_consumer/models.py

```diff
@@ -232,6 +234,29 @@ class LtiConfiguration(models.Model):
         help_text='Enable LTI Proctoring Services',
     )
 
+    @property
+    def is_ccx_location(self):
+        """
+        Check if block location is from a CCX.
+        """
+        return isinstance(self.location, CCXBlockUsageLocator)
+
+    @cached_property
```

Comment:
Why is necessary this property?

Reply:
I used a cached property to avoid querying the database each time I need the LtiConfiguration instance of the master course XBlock: https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.functional.cached_property

Reply:
Nice ! thank you.